### PR TITLE
TA 74385

### PR DESCRIPTION
**Pull request by stbof**

### CODE REVIEW

**Review by mgartner — APPROVED** [review]
LGTM.

**Comment by nickelnickel** [comment]
nit:

change `Corruption can occur under the following conditions:` to `Corruption can only occur under the following conditions:`

File: advisories/a74385.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: Technical Advisory 74385
+summary: Partial indexes can be corrupted by <code>UPDATE</code> statements, resulting in incorrect query results for any queries that use the partial index
+toc: true
+---
+
+Publication date: Jan 6, 2022
+
+## Description
+
+Partial indexes can be corrupted by `UPDATE` statements, resulting in incorrect query results for any queries that use the partial index. Corruption can occur under the following conditions:
```
Comment (stbof) [inline advisories/a74385.md:11]:
```suggestion
Partial indexes can be corrupted by `UPDATE` statements, resulting in incorrect query results for any queries that use the partial index. Corruption can occur only under the following conditions:
```

**Review by nickelnickel — APPROVED** [review]
LGTM

**Comment by stbof** [comment]
> nit:
> 
> change `Corruption can occur under the following conditions:` to `Corruption can only occur under the following conditions:`

Fixed w/ slight usage tweak.

**Review by rmloveland — APPROVED** [review]
LGTM

**Pull request merged**